### PR TITLE
1 separate login and login result structures

### LINKED ISSUE #1: Separate Login and Login Result structures
Use a data structure to hold the user's login request and another data structure to hold the login result.

### PR DESCRIPTION
**Pull request by MayderC**
fix user endpoint

### CODE REVIEW

File: WebApplication3/Helpers/JsonWebToken.cs [hunk 1]
```diff
@@ -17,13 +17,11 @@ public JsonWebToken(string @secretkey)
     }
 
 
-    public string createJWT(User user)
+    public string createJWT(Auth user)
     {
       var claims = new ClaimsIdentity();
 
-      claims.AddClaim(new Claim(ClaimTypes.NameIdentifier, user.name));
-      claims.AddClaim(new Claim(ClaimTypes.NameIdentifier, user.id + ""));
```
Comment (dlacayo-nearlinx) [inline WebApplication3/Helpers/JsonWebToken.cs:25]: Use string interpolation instead of concatenation.

**Review by dlacayo-nearlinx — CHANGES_REQUESTED** [review]
Fix string interpolation

File: WebApplication3/Helpers/JsonWebToken.cs (hunk 1 above)
Comment (MayderC) [reply WebApplication3/Helpers/JsonWebToken.cs:25]: fix, resolve

**Review by dlacayo-nearlinx — APPROVED** [review]

**Comment by dlacayo-nearlinx** [comment]
Looks good. Approved.

**Pull request merged**